lib/workflows/inference_validation/nodes/validate_inferences.py

```diff
@@ -97,7 +130,7 @@ async def _validate_chunk_inferences(
                 ),
                 "paragraph": state.get_paragraph(chunk.paragraph_index),
                 "chunk": chunk.content,
-                "claim": claim.claim,
+                "claim": claim_text,
```

Comment:
There's a reason not to use the claim directly?

Reply:
good call, changed already